logging/kit/ctxkit/context.go

```diff
@@ -0,0 +1,62 @@
+package ctxkit
```

Comment:
To keep parity with other packages would you mind adding in `doc.go` and `examples_test.go` will help with the godoc generation as well.